klarna/ui
# Add ids everywhere

Fixes: https://github.com/klarna/ui/issues/230

## How to review this PR:

1. Pull this branch locally and run `npm start`
2. Go to http://localhost:7777/ui/#UniqueIds

Every component in there should have all the tags marked with id like:

```javascript
<ComponentWithSubelements id='my-component' />

// will become
<div id='my-component'>
  <div id='my-component-sub-element'>
    <h2 id='my-component-title'>Title</h2>
    <p id='my-component-paragraph'>Lorem ipsum dolor sit amet et consequetur</p>
  </div>
</div>
```

Some components will not have an `id` prop specified. These are supposed to take the `id` from the unique `name` prop.
